.github/workflows/ci.yml

```diff
@@ -17,25 +17,49 @@ jobs:
           pre-commit run --all-files || { git diff --color=always && exit 1; }
   unit-tests:
     name: Unit Tests
-    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        os: [ windows-2022, ubuntu-20.04 ]
+        include:
+          - os: windows-2022
+            NEOVIM_CONFIG_PATH: ~/AppData/Local/nvim
+          - os: ubuntu-20.04
+            NEOVIM_CONFIG_PATH: $HOME/.config/nvim
+    runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v2
       - name: Setup Vim
         uses: rhysd/action-setup-vim@v1.2.6
         with:
           neovim: true
       - name: Ensure plugins directory exists
+        env:
+          NEOVIM_CONFIG_PATH: '${{ matrix.NEOVIM_CONFIG_PATH }}'
         run: |
-          mkdir -p $HOME/.config/nvim/plugins
+          mkdir -p $NEOVIM_CONFIG_PATH/plugins
       - name: Create a fake code-minimap executable
         # Necessary for minimap.vim to start up
         run: |
           mkdir -p "$HOME/bin"
           ln -s /bin/echo "$HOME/bin/code-minimap"
           echo "$HOME/bin" >> $GITHUB_PATH
+        if: ${{ !contains(matrix.os, 'windows') }}
```

Comment:
Do these `if` gates have to be at the bottom? It would be nice if it was the first part of the yaml section.

Reply:
I don't think so, I've moved them to the first key in the block. Let's see if GitHub accepts it.